fastify.js

```diff
@@ -259,6 +259,10 @@ function build (options) {
     return fastify
   }
 
+  function exposeRoutes () {
+    return map.entries()
+  }
```

Comment:
I would prefer if we could return an array of objects, rather than an array of arrays.